### PR TITLE
Made test apps disable controls when appcenter is disabled

### PR DESCRIPTION
**Pull request by michaelsmitha**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Puppet app disabling/enabling adjustments

## Related PRs or issues

[AB#64917](https://dev.azure.com/msmobilecenter/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/64917)

### CODE REVIEW

File: Apps/Contoso.WPF.Puppet.DotNetCore/MainWindow.xaml.cs [hunk 1]
```diff
@@ -59,6 +59,8 @@ private void UpdateState()
             AppCenterEnabled.IsChecked = AppCenter.IsEnabledAsync().Result;
             CrashesEnabled.IsChecked = Crashes.IsEnabledAsync().Result;
             AnalyticsEnabled.IsChecked = Analytics.IsEnabledAsync().Result;
+            AnalyticsEnabled.IsEnabled = AppCenterEnabled.IsChecked.Value;
```
Comment (guperrot) [inline Apps/Contoso.WPF.Puppet.DotNetCore/MainWindow.xaml.cs]: We should update the demo ones as well.

File: Apps/Contoso.WPF.Puppet.DotNetCore/MainWindow.xaml.cs [hunk 2]
```diff
@@ -71,7 +73,9 @@ private void AppCenterEnabled_Checked(object sender, RoutedEventArgs e)
 
         private void AnalyticsEnabled_Checked(object sender, RoutedEventArgs e)
         {
-            if (AnalyticsEnabled.IsChecked.HasValue)
+            var isAppCenterEnabled = AppCenterEnabled.IsChecked.Value;
+            AnalyticsEnabled.IsEnabled = isAppCenterEnabled;
+            if (AnalyticsEnabled.IsChecked.HasValue && isAppCenterEnabled)
```
Comment (guperrot) [inline Apps/Contoso.WPF.Puppet.DotNetCore/MainWindow.xaml.cs]: The check is done by SDK

File: Apps/Contoso.WPF.Puppet.DotNetCore/MainWindow.xaml.cs (hunk 1 above)
Comment (guperrot) [reply Apps/Contoso.WPF.Puppet.DotNetCore/MainWindow.xaml.cs]: (and we have 2 new demo apps now on develop)

**Comment by michaelsmitha** [comment]
@guperrot comments resolved

**Pull request merged**